### PR TITLE
Release pwa-extension plugin version 1.0.3

### LINKED ISSUE #1: Utilize site icon customizer option for PWA icon
Current code asks for separate `/assets/img/icon-%2$sx%2$s.png` inside theme. We can utilize site icon itself to generate app icon sizes. 

Goal is to make manifest file ready for all WordPress sites, and with just site icon add-to-homescreen feature can be enabled. 

- [x] Use site icon to generate PWA app icon sizes - https://github.com/rtCamp/pwa-extension/blob/master/class-pwa-manifest.php#L32-L35
- [ ] Add filter for App Icon URLs so that we can filter Icon URLs if needed.

### LINKED ISSUE #33: Conflicts with iZooto plugin
```
Test Environment:
Plugins active:
Official PWA plugin: v0.3.0
rt PWA Extension: v1.0.2
iZooto plugin: v3.6.6
```
1. Offline reading is working in mobile :heavy_check_mark: and while checking in desktop :heavy_multiplication_x: faced errors in the console. Please check the screenshot.
Screenshot from 2020-03-09 10-33-04

2. Add to Home Screen - Add to home screen option was not visible in mobile :heavy_multiplication_x: . On desktop the install option is shown on the browser bar :heavy_check_mark: . On mobile tried clearing cache but still the add to home screen option was not visible. Note: http auth is been removed from the site 

3. iZooto Push Notifications - Push notifications are working in both mobile and desktop. Created Post and enabled notification option on post publish/update notifications are triggered on mobile and desktop. 

**Note: 
The above 1st and 2nd issues occurs when iZooto pluign is enabled. If this plugin is disabled then above mentioned issues works fine.
iZooto plugin is creating conflicts with the other features, offline reading and Add to Home screen.**

### LINKED ISSUE #41: Gravity form submission does not work if the current form URL is added in the menu
Tracking this https://github.com/rtCamp/pwa-extension/pull/18#issuecomment-609707217 here

The Gravity form submission does not work if the form page is part of the menu. When the menu precache tries to precache the form URL the does not submit and just redirects to the current page.

Version: 1.0.3(Develop)

Ways to reproduce: 
1) Create a page with gravity form.
2) Add page to the menu 
3) Try to submit the gravity form on that page it will redirect to the same page.
4) Remove the page from the menu and the submission works fine for that form.

### PR DESCRIPTION
**Pull request by deepaklalwani97**
This PR includes the following updates:-
Tested Upto 0.5.0 version of the [PWA](https://wordpress.org/plugins/pwa/) plugin
- Add unit test cases for plugin #20 (Fixes #20 ).
- Add gravity form offline submission support #18.
- Add filter to change the app icon images #23. ( Fixes #1  )
- Fixed conflict with izooto plugin and gravity form offline submission service worker script #36.( Fixes #33 )
- Removed menu and recent blog pre caching to fix gravity form submission conflict. #42 ( Fixes #41  )

### CODE REVIEW

**Review by sagarnasit — DISMISSED** [review]

**Review by sagarnasit — DISMISSED** [review]

**Comment by pradeep910** [comment]
I have done minor Readme updates. 

Can we release this now? @sagarnasit @deepaklalwani97

**Comment by sagarnasit** [comment]
@pradeep910 There is some issue with `contact form` submission in the new update of PWA plugin.

**Comment by juhi123** [comment]
@sagarnasit I have cross checked this issues https://github.com/rtCamp/pwa-extension/pull/18#issuecomment-609707217, it's still not working

**Comment by deepaklalwani97** [comment]
@juhi123 Can you please perform one round of QA testing on the develop branch before we release it to master.

**Comment by juhi123** [comment]
@deepaklalwani97 Done with testing, looks fine 👍

File: README.md [hunk 1]
```diff
@@ -17,23 +17,89 @@ An extension to [PWA](https://wordpress.org/plugins/pwa/) to enable `add to home
...
+
+1. Push the code changes from your local clone to your fork.
+
+1. Open a pull request and that's it! We'll with feedback as soon as possible (Isn't collaboration a great thing? 😌)
+
+1. Once your pull request has passed final code review and tests, it will be merged into `develop` and be in the pipeline for the next release. Props to you! 🎉
+
 ## Change Log
 
+### v1.0.3 (4-02-2020)
```
Comment (sagarnasit) [inline README.md]: We reverted the last release, so, please update it with the latest date.

File: README.md [hunk 2]
```diff
@@ -44,9 +110,19 @@ An extension to [PWA](https://wordpress.org/plugins/pwa/) to enable `add to home
 
 ### v1.0.0 (12-09-2019)
 
-- `Add to Home Screen` popup for mobiles with custom icon setup in WP theme `(your-theme//assets/img/icon-{width x height}.png'`.
+- `Add to Home Screen` popup for mobiles with custom icon setup in WP theme `(<your-theme>/assets/img/icon-{width}x{height}.png'`.
```
Comment (sagarnasit) [inline README.md:113]: Please add missing ending brackate `)`.
```suggestion
- `Add to Home Screen` popup for mobiles with custom icon setup in WP theme `(<your-theme>/assets/img/icon-{width}x{height}.png')`.
```

**Review by sagarnasit — DISMISSED** [review]
@deepaklalwani97 Approving PR with a couple of minor change requests. Also, make sure Gutenberg asset caching doesn't create a problem on WP or Gutenberg plugin update, Make sure to pass this test case to QAs on their testing list. Everything else looks good to me :)

**Comment by deepaklalwani97** [comment]
Thanks, @sagarnasit I have updated the readme and as for caching Gutenberg assets is concerned it will not create any issues as we are using network first strategy for caching the assets so whenever the network is available it would fetch assets from the network first and update the cache.

**Review by sagarnasit — APPROVED** [review]

**Pull request merged**